### PR TITLE
Added simple transformers 'FirstArrayElement' and 'LastArrayElement

### PR DESCRIPTION
**Pull request by tundisto**
Added shortcut transformer 'FirstArrayElement'.
Added shortcut transformer 'LastArrayElement'.

## Status
Ready

## Related Issues
N/A

## Description
Adds two simple new transformers, namely 'FirstArrayElement' and 'LastArrayElement'.  I realise that we already have the index transformer, but for code noobs these would be helpful. IMHO.

## Screenshots
image
image

## Related PRs
N/A

## Required version of Demisto
4.x/5.x

## Does it break backward compatibility?
No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
None

## Additional changes
None

### CODE REVIEW

**Comment by Itay4** [comment]
@tundisto please convert the scripts to directory package format

**Comment by tundisto** [comment]
Converted to package format and unit tests added.  Is there anything else needed?

**Comment by Itay4** [comment]
@kirbles19 please review:
https://github.com/demisto/content/blob/392033c82d48d88fdedce976aa99ddaa6d5ad946/Scripts/FirstArrayElement/FirstArrayElement.yml
https://github.com/demisto/content/blob/392033c82d48d88fdedce976aa99ddaa6d5ad946/Scripts/LastArrayElement/LastArrayElement.yml

**Comment by Itay4** [comment]
@tundisto please note [the build](https://circleci.com/gh/demisto/content/31329) fails

**Comment by tundisto** [comment]
It says "There is no test-playbook that checks the changes you've done". Does that mean an actual playbook or a unit test?

**Comment by Itay4** [comment]
> It says "There is no test-playbook that checks the changes you've done". Does that mean an actual playbook or a unit test?

that refers to a test playbook, but a unit test should be fine in this case
just need to mention it under `tests` in the yml with something like `- No test - unit test`

**Comment by tundisto** [comment]
Build is now passing

File: Scripts/LastArrayElement/LastArrayElement.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import demistomock as demisto
+
+VALUE = demisto.args()['value']
```
Comment (Itay4) [inline Scripts/LastArrayElement/LastArrayElement.py]: same comments here

**Comment by kirbles19** [comment]
@tundisto @Itay4 Done.

**Comment by tundisto** [comment]
Unit tests submitted and build is passing.

File: Scripts/FirstArrayElement/FirstArrayElement.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import demistomock as demisto
+
+VALUE = demisto.args()['value']
```
[the remark this answers is not in the document]
Comment (tundisto) [reply Scripts/FirstArrayElement/FirstArrayElement.py]: I changed 'value' to lowercase in the source

File: Scripts/FirstArrayElement/FirstArrayElement.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+if type(VALUE) is list and len(VALUE) > 0:
```
[the remark this answers is not in the document]
Comment (tundisto) [reply Scripts/FirstArrayElement/FirstArrayElement.py]: I changed 'value' to lowercase in the source

File: Scripts/LastArrayElement/LastArrayElement.py (hunk 1 above)
Comment (tundisto) [reply Scripts/LastArrayElement/LastArrayElement.py]: I changed 'value' to lowercase in the source

**Pull request merged**